docs/glossary.md

```diff
@@ -42,19 +45,22 @@ ballot measure, that appears on the ballot for an election in a
 county.
 
 * **ballot manifest** - A document that describes how ballots are
-organized and stored, and relates a CVR (defined below) to the
+organized and stored, and relates a Cast Vote Record to the
 physical location in which the original marked ballot is stored. The
-ballot manifest should show, in human-readable format, the physical
-location of a ballot to allow a county worker to efficiently find the
+ballot manifest specifies the physical
+location of a ballot to allow staff to find the
 specific ballot represented by a given CVR. A ballot manifest will
 contain the following information: county ID, tabulator ID, batch ID,
-(Shown in Appendix 2.) number of ballots in each batch, and the
+the number of ballots in each batch, and the
 storage location where the batch is secured following tabulation. A
-sample ballot manifest is attached as Appendix 2.
+sample ballot manifest is provided at `test/manifest-dq.csv`.
 
-* **cast vote records (CVR)** - Electronic records created during
-scanning of paper ballots indicating how the voting system interpreted
-physical marks on every ballot. A sample CVR is attached as Appendix 1.
+* **cast vote record (CVR)** - An electronic record
+indicating how the marks on a ballot were interpreted as votes.
+May be created by a scanner or DRE, or manually during an audit.
+Sample CVRs in Dominion's format are in
+`test/dominion-2017-CVR_Export_20170310104116.csv`.
```

Comment:
Per the earlier discussion, we should put this file in `docs/samples/` too, right?